decidim-budgets/spec/commands/decidim/budgets/admin/import_proposals_to_budgets_spec.rb

```diff
@@ -64,6 +66,14 @@ module Admin
               end.to change { Project.where(budget: budget).count }.by(1)
             end
 
+            context "when there are no proposals in the selected scope" do
+              it "doesn't create any project" do
+                expect do
+                  command.call
+                end.not_to(change { Project.where(budget: budget).where(scope: scope).count })
```

Comment:
I'm not sure I'm understanding this test.

You've created a new, scope, but you are not using it anywere, and the `let` is without `!` so it never gets created. The form double is initialized with `scope_id: ""` instead of `scope`, and also the name of the form attribute is `decidim_scope_id`, not `scope_id` so not sure if this is being setted correctly.
In the end the command executes `all_proposals.where(decidim_scope_id: form.scope_id)` and filters proposals were the `scope_id` equals an empty string, instead of the scope that was introduced for testing this situation but never gets created.

So I think the test succeeds accidentally but not because it executes as expected.
can you please give it a check and clarify my mind :laughing:

Reply:
Don't worry, this is actually super helpful! 😄  Let me check this so I can properly answer you all, guys 🙏🏻

Reply:
@Leusev @tramuntanal Ok, I didn't update all instances [when renaming](https://github.com/decidim/decidim/pull/6525/commits/8273a8c722a002ec19fbb29fde355e70cd14fa63) `decidim_scope_id` to `scope_id`.
I'll fix this 🙏🏻 TY again